### ISSUE TITLE
Port importer logic to exporter

### ISSUE THREAD

**Issue by sdorokhova** [camunda/camunda#18304]
### Description

To start with converting importer to exporter, we need to copy the current importer logic to new Zeebe exporter. It must not be ideal and take into account all the changes that we plan to perform within this epic, but rather do exactly the same as importer is doing now.

### TODO

- [x] create new Zeebe exporter module: PR #18886
- [x] implement simple code structure to transform Zeebe records to Operate entities and persist them in Operate indices: the same PR #18886
- [x] implement BatchRequest using the new ES client https://github.com/camunda/camunda/issues/19108

Copy all business logic from `*ZeebeRecordProcessor` classes to use in record transformation: 
- [x] #18908
- [x] #18911
- [x] #18912
- [x] #18913
- [x] #18914
- [x] #18915
- [x] #19387
- [x] #22642

### Additional Information

* performance is not a priority in this issue
* [prototype code](https://github.com/camunda/operate/issues/6031) can be used when needed, but importer logic must be copied from current `main`

---------------------------------------------------------------------------------------------

### Acceptance Criteria

- [ ] Checked if the specific data needs to be provided for the QA process

### Definition of Ready - Checklist

- [ ] The issue has a meaningful title, description and testable acceptance criteria
- [ ] If documentation needs to be updated, an issue is created in the [camunda-docs](https://github.com/camunda/camunda-docs) repo, and the issue is added to our Operate project board.
- [ ] If HELM charts need to be updated, an issue is created in the [camunda-platform-heml](https://github.com/camunda/camunda-platform-helm) repo, and the issue is added to our Operate project board.
- [ ] Potential Security risks have been considered
- [ ] Cross-team dependencies have been considered

**Comment by sdorokhova** [comment]
I move this task back in ready state, but I will pick it right after #19006 that I'm working on now.

**Comment by Zelldon** [comment]
Everything basic was done for the MVP. The complex parts are part of https://github.com/camunda/camunda/issues/24098. I will close this issue then
